src/client/api/event.c

```diff
@@ -1028,6 +1028,10 @@ daos_event_init(struct daos_event *ev, daos_handle_t eqh,
 		evx->evx_sched = &eqx->eqx_sched;
 		daos_eq_putref(eqx);
 	} else {
+		if(daos_sched_g.ds_udata == NULL) {
```

Comment:
(style)  space required before the open parenthesis '('

Reply:
Done